src/System.Net.Http/tests/UnitTests/HPack/HPackRoundtripTests.cs

```diff
@@ -9,27 +9,17 @@
 using System.Net.Http.HPack;
 using System.Net.Http.Headers;
 using System.Text;
+using System.Web;
```

Comment:
This is bringing in System.Web for `HttpUtility` class?

Why not use System.Net.WebUtility instead?

Reply:
fixed